tests/testthat/test-derive_var_obs_number.R

```diff
@@ -1,58 +1,167 @@
-test_that("first observation for each group are selected", {
+## Test 1: create default new variable ASEQ ----
+test_that("derive_var_obs_number Test 1: create default new variable ASEQ", {
   input <- tibble::tribble(
-    ~USUBJID, ~AVISITN, ~AVAL,
-    1, 1, 12,
-    1, 3, 9,
-    2, 2, 42,
-    3, 3, 14,
-    3, 3, 10
+    ~STUDYID, ~USUBJID, ~VSTESTCD, ~VISITNUM, ~VSTPTNUM,
+    "TEST01",  "PAT01",   "DIABP",        1L,        1L,
+    "TEST01",  "PAT01",   "PULSE",        1L,        1L,
+    "TEST01",  "PAT01",   "PULSE",        1L,        2L,
+    "TEST01",  "PAT01",   "PULSE",        2L,        1L,
+    "TEST01",  "PAT01",   "PULSE",        2L,        2L,
+    "TEST01",  "PAT01",   "SYSBP",        1L,        1L,
+    "TEST01",  "PAT01",   "SYSBP",        1L,        2L,
+    "TEST01",  "PAT01",   "SYSBP",        2L,        1L,
+    "TEST01",  "PAT02",   "DIABP",        1L,        1L,
+    "TEST01",  "PAT02",   "DIABP",        2L,        1L,
```

Comment:
We haven't used the integer `L` in previous tests.  Could you update one of the tests to not use it please

Reply:
updated